test/fixtures/templates/deliveries/_form.html.erb

```diff
@@ -2,7 +2,7 @@
   form_for(form, class: 'form-horizontal') do
     div class: 'form-group' do
       label :customer
-      input_text :customer, class: 'form-control', placeholder: 'Customer', name: nil
+      input_text :customer, class: 'form-control', placeholder: 'Customer', name: nil, required: true
```

Comment:
I think testing it through integration test is overkill, can you please add it to `form_helper_test`'s `text_field` instead?

Something like:

```
    it "allows to specify required constraint" do
      actual = view.form_for(:book, action) do
        text_field :title, required: true
      end.to_s

      actual.must_include %(<input type="text" name="book[title]" id="book-title" required">)
    end
```